tests/test_save_load.py

```diff
@@ -1,5 +1,6 @@
 import os
```

Comment:
If saving to `io.BytesIO` (or any IOBase) is supported, that should be tested. This is probably easy copy paste from [stable-baselines tests](https://github.com/hill-a/stable-baselines/blob/master/tests/test_save.py#L40).